### PR TITLE
 systeroid: init at 0.1.1 

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/kernel/htmldocs.nix [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  installPhase = ''
+    mkdir -p $out/share/doc
+    mv Documentation/output $out/share/doc/linux-doc
+    cp -r Documentation/* $out/share/doc/linux-doc/
+  '';
+
+  meta = with lib; {
+    description = "Linux kernel html documentation";
+    platforms = platforms.linux;
```
Comment (dotlambda) [inline pkgs/os-specific/linux/kernel/htmldocs.nix:79]: `homepage`, `license`, and `maintainers` missing

File: pkgs/os-specific/linux/kernel/htmldocs.nix [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      sphinx_rtd_theme = prev.sphinx_rtd_theme.override {
+        inherit sphinx;
+        docutils = docutils_old;
+      };
+    };
+  };
+in
+
+stdenv.mkDerivation {
+  name = "test";
```
Comment (dotlambda) [inline pkgs/os-specific/linux/kernel/htmldocs.nix:42]: Use an appropriate `pname` and specify `version`.

File: pkgs/os-specific/linux/kernel/htmldocs.nix [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+, linux_latest
+, makeFontsConf
+, perl
+, python310
+, sphinx
+, which
+}:
+
+let
+  py = python310.override {
```
Comment (dotlambda) [inline pkgs/os-specific/linux/kernel/htmldocs.nix:14]:
```suggestion
  py = python3.override {
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by vcunat** [comment]
Why these version overrides inside `linux-doc`?  Removing them would fix the builds of these two packages in current `staging-next` (PR #216499)  Failure on Hydra: https://hydra.nixos.org/build/210096444

**Pull request merged**